docs/DOCS_README.md

```diff
@@ -20,7 +20,8 @@ a private website repository has make targets consumed by a standard Jenkins tas
 ## README
 
 The [README.md](./README.md) is also the landing page for the documentation
-on the website.
+on the website.i During the Jenkins build, the current commit is added to the bottom
```

Comment:
`.i`

Reply:
I removed `.i`, hope it wasn't intentional for some strange reason